### PR TITLE
Add getValue() for Rational and Integer (GMP and CLN).

### PR DESCRIPTION
**Pull request by aniemetz**
Returns a copy of d_value to enable public access of GMP and CLN data.

### CODE REVIEW

File: src/util/integer_gmp_imp.h [hunk 1]
```diff
@@ -78,6 +78,14 @@ class CVC4_PUBLIC Integer {
 
   ~Integer() {}
 
+  /**
+   * Returns a copy of d_value to enable public access of GMP data.
+   */
+  mpz_class get_mpz_copy()
+  {
+    return mpz_class(d_value.get_mpz_t());
```
Comment (timothy-king) [inline src/util/integer_gmp_imp.h]: Does it suffice to just "return d_value;"?

Comment (aniemetz) [reply src/util/integer_gmp_imp.h]: Indeed, it is. Renamed to getValue() and added a CLN implementation.

File: src/util/integer_cln_imp.h [hunk 2]
```diff
@@ -102,6 +102,14 @@ class CVC4_PUBLIC Integer {
 
   ~Integer() {}
 
+  /**
+   * Returns a copy of d_value to enable public access of CLN data.
+   */
+  cln::cl_I getValue()
```
Comment (4tXJ7f) [inline src/util/integer_cln_imp.h]: Please change the type of this method and the other methods  `cln::cl_I getValue() const` to indicate that none of the class fields are modified.

**Comment by aniemetz** [comment]
@4tXJ7f  done

**Pull request merged**